pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarClientImpl.java

```diff
@@ -884,9 +884,9 @@ public void reloadLookUp() throws PulsarClientException {
             TopicName topicName = TopicName.get(topic);
             AtomicLong opTimeoutMs = new AtomicLong(conf.getLookupTimeoutMs());
             Backoff backoff = new BackoffBuilder()
-                    .setInitialTime(100, TimeUnit.MILLISECONDS)
+                    .setInitialTime(conf.getInitialBackoffIntervalNanos, TimeUnit.NANOSECONDS)
```

Comment:
Nice catch, looks like we apply the configuration for producer and consumer but missed the client part.

Could you please help add a test for covering the changes to make sure we will not make a regression in the future?

Reply:
@codelipenghui - what would you like to see tested? I think it is pretty well defined to use the config for the backoff here. My only concern would be backwards compatibility, but I don't see that being a real issue. The default config here is 100 milliseconds.

Reply:
@michaeljmarshall I mean to add a test to cover the backoff interval is configurable.

Reply:
@codelipenghui - thanks for clarifying. I'm not exactly sure how to add that test, but I think we could probably merge this as is.